src/ser.rs

```diff
@@ -202,13 +211,19 @@ impl ser::SerializeStruct for ObjectSerializer<'_> {
 /// A [`serde::Serializer`] that converts supported Rust values into a [`JsValue`].
 // Serializer might be configurable in the future, so add but hide its implementation details.
 #[derive(Default)]
-pub struct Serializer(());
+pub struct Serializer {
+    serialize_maps_as_objects: bool,
+}
 
 impl Serializer {
     /// Creates a new default [`Serializer`].
     pub fn new() -> Self {
         Default::default()
     }
+
+    pub fn serialize_maps_as_objects(&mut self, value: bool) {
```

Comment:
Should this rather be
```rust
pub fn serialize_maps_as_objects(mut self, value: bool) -> Self {
```
so that the call can be chained onto `::new()` ?

Reply:
I think that would be better, yeah.

Reply:
Please remember to add a doc comment though.